Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -373,7 +373,7 @@ def ap_find_tests(bld, use=[]):
 
 @conf
 def ap_version_append_str(ctx, k, v):
-    ctx.env['AP_VERSION_ITEMS'] += [(k, '"{}"'.format(os.environ.get(k, v)))]
+    ctx.env['AP_VERSION_ITEMS'] += [(k, '"{}"'.format(os.environ.get(k) if k in os.environ else v)))]
```

Comment:
Isn't this a noop?  There's no evaluation going on here as you've described in the PR.

Reply:
Yup my bad. Moving this up to the `ap_version_append_str` call.